README.md

```diff
@@ -1,5 +1,9 @@
 # Operating Systems (OER)
 
+[![](https://img.shields.io/static/v1?label=license&message=CCBY-SA4.0&color=%23385177)](https://github.com/open-education-hub/operating-systems/blob/master/COPYING.md)
```

Comment:
This generated `markdownlint` errors. Try to add a placeholder or ignore `MD045` in this file.

```suggestion
[![copying.md](https://img.shields.io/static/v1?label=license&message=CCBY-SA4.0&color=%23385177)](https://github.com/open-education-hub/operating-systems/blob/master/COPYING.md)
```